.github/workflows/test.yml

```
@@ -59,6 +59,8 @@ jobs:
           New-Item -ItemType Directory -Force target | Out-Null
           rustc --edition=2021 --test src-tauri/tests/home_resolution_contract.rs -o target/home-resolution-contract.exe
           & .\target\home-resolution-contract.exe
+      - name: Windows photo open-handle identity regression
+        run: cargo test --manifest-path src-tauri/Cargo.toml photo_duplicate::tests:: --lib
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)